build/README.md

```diff
@@ -0,0 +1,19 @@
+# SIG Doc builder
+
+This script will generate the following documentation files:
+
+```
+sig-*/README.md
+sig_list.md
+```
+
+Based off the `sigs.yaml` metadata file.
+
+## How to use 
+
+To (re)build documentation for all the SIGs, run these commands:
+
+```bash
+docker build -t sigdocs -c build/Dockerfile build
+docker run -v $(pwd):/go/src/app sigdocs
```

Comment:
Looks like it works for me now. I thought I had tried it last time and saw that it didn't, but maybe that's because the build didn't work properly.